ci-operator/templates/openshift/openshift-azure/build-node-image-3.10.yaml

```diff
@@ -0,0 +1,351 @@
+# This template is being used for testing OpenShift Ansible image builds for Azure.
```

Comment:
I was hoping we can reuse the existing template. Why is this necessary?

Reply:
Hm.. You might be right, let me check